flocker/node/_docker.py

```diff
@@ -710,8 +693,7 @@ def _remove():
                 # attempting removal or use -f")'
                 # This code should probably be removed once the above
                 # issue has been resolved. See [FLOC-1850]
-                while self._blocking_container_runs(container_name):
-                    sleep(0.01)
+                self._client.wait(container_name)
```

Comment:
:question: The description of this mentions a 404 error, but the issue comments describe that it is a 500 error now.